libs/resolver/workspace.rs

```diff
@@ -942,8 +970,12 @@ impl<TSys: FsMetadata + FsRead> WorkspaceResolver<TSys> {
       })
       .collect::<BTreeMap<_, _>>();
     let fs = CachedMetadataFs::new(sys, fs_cache_options);
-    let sloppy_imports_resolver =
-      new_rc(SloppyImportsResolver::new(fs, sloppy_imports_options));
+    let compiler_options_resolver = CompilerOptionsResolverCell::default();
```

Comment:
Nitpick: maybe `CompilerOptionsResolverCellRc` for clarity that it's in an arc/rc